unstoppable-ios-app/domains-manager-ios/Extensions/Extension-String+Preview.swift

```diff
@@ -1232,6 +1232,7 @@ extension String {
         static let removeMPCWalletPullUpSubtitle = "REMOVE_MPC_WALLET_PULL_UP_SUBTITLE"
         static let mpcWalletMessagingUnavailableMessage = "MPC_WALLET_MESSAGING_UNAVAILABLE_MESSAGE"
         static let mpcWalletSigningUnavailableErrorMessage = "MPC_WALLET_SIGNING_UNAVAILABLE_ERROR_MESSAGE"
+        static let mpcWalletDefaultName = "MPC_WALLET_DEFAULT_NAME"
```

Comment:
Nobody knows what default is) How about 'mpcWalletShortName' -- speaks for itself

Reply:
This is specifically the string that will be used as default name for the newly added MPC wallet. Default makes a lot of sense here. 
On the other hand mpc wallet short name implies that there's long and there's short name. MPC product name is not the same is mpc wallet name. 
And mpcWalletShortName might not scale well when product name will be changed.